src/Greeting.jsx

```diff
@@ -0,0 +1,25 @@
+// import React from 'react';
+
+// function Greeting({ username }) {
+//   return (
+//     <div>
+//       <h1>Hello, {username}!</h1>
+//     </div>
+//   );
+// }
+
+// export default Greeting;
```

Comment:
[OPTIONAL]
- [ ] Please remove all commented out code from your code base. It make your project look messy and unclean